internal/collector/persistentvolume.go

```diff
@@ -108,6 +108,21 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_persistentvolume_capacity",
+			Type: metric.MetricTypeGauge,
+			Help: "persistentvolume capacity in Gigabytes.",
```

Comment:
```suggestion
			Help: "Persistentvolume capacity in gigabytes.",
```

Plus the suggestion by @brancz.